src/components/containers/SingleArticle.jsx

```diff
@@ -168,7 +171,7 @@ class SingleArticle extends Component {
                       : () => this.displayForm('login')
                   }
                 />
-                <CommentIconComponent className="reaction-logo" commentCount={0} />
+                <CommentIconComponent className="reaction-logo" commentCount={this.props.comments.length} />
```

Comment:
comments can be destructured from this.props since it is used again on line 205.